wlauto/workloads/multiapp/__init__.py

```diff
@@ -61,17 +61,15 @@ class Multiapp(AndroidUiAutoBenchmark):
     For reliable testing, this workload requires a good and stable internet connection,
     preferably on Wi-Fi.
 
+    NOTE: This workload requires two jpeg files to be placed in the
```

Comment:
I found I needed four images otherwise wa-working view element doesn't get placed at the top of the gallery view.